### ISSUE TITLE
Enter Edit mode by clicking

### ISSUE THREAD

**Issue by BobTB** [Laverna/laverna#829]
Edit button is just not needed, just clicking on a note should open it up in edit mode, simpler and faster. If I can press "e" to edit it, why not enable mouse click too.

**Comment by klenium** [comment]
I disagree. I often need to copy long texts from my note. To do that, I need to select the text by clicking on it 2 times. Auto edit on click vent would be very annoying.

**Comment by BobTB** [comment]
ok. I dont understand. For example windows Notepad. Clicking and selecting and copy and stuff, no need for any edit button. Edit works if I type something, selecting text is not something to interfere with edit. Check standardnotes (https://standardnotes.org/)  or boostnote (https://boostnote.io).
